fw/http_frame.h

```diff
@@ -228,6 +228,7 @@ int tfw_h2_context_init(TfwH2Ctx *ctx);
 void tfw_h2_context_clear(TfwH2Ctx *ctx);
 int tfw_h2_frame_process(TfwConn *c, struct sk_buff *skb);
 void tfw_h2_conn_streams_cleanup(TfwH2Ctx *ctx);
+TfwStream *tfw_h2_find_stream_if_not_closed(TfwH2Ctx *ctx, unsigned int id);
```

Comment:
It's not mandatory, but I suggest to renaming `tfw_h2_find_stream_if_not_closed` to `tfw_h2_find_stream_not_closed` or to `tfw_h2_find_not_closed_stream`

Reply:
yes, fixed